src/RoboFile/ClientContainer.php

```diff
@@ -42,7 +42,24 @@ public function runContainerTests(
 		$opts['install-test'] = '';
 
 		$this->runContainerTestPreparation($opts);
+
 		$this->runTestSuites($opts);
+
+	}
+
+	public function runContainerTest(
```

Comment:
I've left runTestSingle there because its implementation generates the acceptance yml config for every test run, according to the specified server.